.pep8speaks.yml

```diff
@@ -0,0 +1,22 @@
+# File : .pep8speaks.yml
+
+message:  # Customize the comment made by the bot
+    opened:  # Messages when a new PR is submitted
+        header: "Hi @{name}, thanks for submitting the PR!"
+                # The keyword {name} is converted into the author's username
+        footer: "Check out the [Hitchhiker's guide to code style](http://docs.python-guide.org/en/latest/writing/style/)!"
+                # The messages can be written as they would over GitHub
+    updated:  # Messages when new commits are added to the PR
+        header: "Hi @{name}, thanks for updating the PR!"
+        footer: ""  # Why to comment the link to the style guide everytime? :)
+    no_errors: "There are no PEP8 issues."
+
+scanner:
+    diff_only: False  # If True, errors caused by only the patch are shown
+
+pycodestyle:
+    max-line-length: 79  # Default is 79 in PEP8
+    ignore:  # Errors and warnings to ignore
```

Comment:
I guess there are some in our settings?

Reply:
We have one in our settings, but it doesn't seem to apply anymore. It's E402:
```
relative-beyond-top-level (E0402): *Attempted relative import beyond top-level package*
```
It's not triggered anymore as it seems, so I'd just leave this section empty for now.